package.json

```diff
@@ -68,6 +68,7 @@
     "lint": "grunt lint",
     "outdated": "npm outdated --depth 0",
     "postinstall": "bower update --config.interactive=false -s",
+    "scss-lint": "scss-lint app/styles || true",
```

Comment:
The `|| true` hackery is because I didn't really care about [exit status codes](https://github.com/causes/scss-lint#exit-status-codes).